boltdesignsystem/bolt
# DS-147: Add Elements and Layouts to PL docs-site

## Jira

https://pegadigitalit.atlassian.net/browse/DS-147

## Summary

Add Elements and Layouts to PL docs-site.

## Details
Rename directories in PL from:
```
01-visual-styles
02-components
04-pages
06-experiments
```

to:
```
10-visual-styles
_20-elements (hidden)
_30-layouts (hidden)
40-components
50-pages
999-experiments
```

Also, did a global find/replace to update where these paths appeared, mostly in component tests.

> Note: this doesn't change the short-links to each section, e.g. `/pattern-lab/?p=viewall-pages-full-page-theming`, but it _does_ change the full path to each page itself when it is not inside the PL iframe.

## How to test

Checkout branch locally, run `yarn clean` and `yarn setup`, or view feature branch when it's deployed.

Verify PL demo pages work as expected.
